changelog.txt

```diff
@@ -1,5 +1,8 @@
 This file contains old changelogs and it's refreshed periodically.
 
+= 3.1.2 =
```

Comment:
Ahh, this is not a good place for changelog! This is an old, archived changelog. The most recent changes are listed in `readme.txt` file.

And when you are starting new section you are not incrementing the version, so instead `= 3.2.1 =` you do `= [Next] =`. This is what we parse during the release.